code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -101,9 +101,19 @@
 	return
 
 /turf/open/floor/plating/asteroid/singularity_act()
+	src.ChangeTurf(/turf/open/space)
```

Comment:
@KorPhaeron Forgot about that, fixed it now. 

Also there's other issues with singularities on lavaland, such as /mob/living/simple_animal/hostile/megafauna not being gibbed by them.